README.md

```diff
@@ -40,29 +40,51 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Check out repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
       - name: Set up MATLAB
-        uses: matlab-actions/setup-matlab@v1
+        uses: matlab-actions/setup-matlab@v2-beta
       - name: Run tests and generate artifacts
         uses: matlab-actions/run-tests@v1
         with:
           test-results-junit: test-results/results.xml
           code-coverage-cobertura: code-coverage/coverage.xml
 ```
 
+### Run MATLAB Script on GitHub-Hosted Runner
+Set up a GitHub-hosted runner to run the commands in a file named `myscript.m` in the root of your repository. To run the script, include the [Run MATLAB Command](https://github.com/matlab-actions/run-command/) action in your workflow.
+
+```yaml
+name: Run MATLAB Script on GitHub-Hosted Runner
+on: [push]
+jobs:
+  my-job:
+    name: Run MATLAB Script
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+      - name: Set up MATLAB
+        uses: matlab-actions/setup-matlab@v2-beta
+      - name: Run script
+        uses: matlab-actions/run-command@v1
+        with:
+          command: myscript
+```
+
 ## Set Up MATLAB
 When you define your workflow in the `.github/workflows` directory of your repository, specify the **Setup MATLAB** action as `matlab-actions/setup-matlab@v1`. The action accepts an optional input.
 
 | Input     | Description |
 |-----------|-------------|
-| `release` | (Optional) MATLAB release to set up. You can specify R2020a or a later release. If you do not specify `release`, the action sets up the latest release of MATLAB.<br/>**Example**: `R2021a`
-
+| `release` | (Optional) MATLAB release to set up. You can specify R2020a or a later release. If you do not specify `release`, the action sets up the latest release of MATLAB.<br/>**Example**: `R2022a`
+| `products` | (Optional) List of products to install using product names separated by spaces. Replace spaces within names with underscores. [See the MathWorks website](https://www.mathworks.com/products.html) for the full list of available products and their names.<br/> **Example**: `MATLAB Simulink Deep_Learning_Toolbox Parallel_Computing_Toolbox`
```

Comment:
Could you install PCT only? Will PCT install MATLAB too? Should this be clarified?

Reply:
It always installs MATLAB and PCT no matter what, but the user is allowed to specify them in this list as well. When we move from the initial "beta" release to our official "v2.0.0" it will always install MATLAB in all cases, but will no longer install PCT unless it is specified.

Reply:
Ok, should it be clarified? By default, the task installs MATLAB and the Parallel Computing Toolbox?